ConsentViewController/Classes/SourcePointClient/SourcepointClientCoordinator.swift

```diff
@@ -698,21 +716,21 @@ class SourcepointClientCoordinator: SPClientCoordinator {
 
     func pvData(pubData: SPPublisherData?, _ handler: @escaping () -> Void) {
         let pvDataGroup = DispatchGroup()
-        if let gdprMetadata = state.gdprMetaData {
+        if let gdprMetadata = state.gdprMetaData, campaigns.gdpr != nil {
```

Comment:
Is this to filter campaign?

Reply:
Yes. To give you more context, the SDK stores an object called `gdprMetaData` in the response of `/meta-data`. But we never clear those objects. So, if in the past, the got a response for GDPR, but the config no longer has a GDPR campaign, we would still have a `gdprMetaData` object stored. 

This change basically checks if we have the `gdprMetaData` stored _and_ if gdpr is part of the config. 

Same for CCPA and USNat